src/Spec.ts

```diff
@@ -147,6 +147,10 @@ export default class Spec {
       this.opts.copyright = true;
     }
 
+    if (!this.opts.hasOwnProperty('ecma262Biblio')) {
```

Comment:
wouldn't this break if someone passed a `hasOwnProperty` op?
```suggestion
    if (!Object.prototype.hasOwnProperty.call(this.opts, 'ecma262Biblio')) {
```

Reply:
Hmm, I was just copying the style of the surrounding code though.

You are right that calling `ecmarkup` with `--hasOwnProperty foo` does give:
```
TypeError: this.opts.hasOwnProperty is not a function
    at new Spec (/Users/rkirsling/Documents/GitHub/ecmarkup/lib/Spec.js:75:24)
```
...but then it's a pre-existing issue.

I can file an issue to have ecmarkup properly guard against unexpected options across the board?

Reply:
no need for an issue; but sure, a separate PR to fix all such cases, with tests, seems great

Reply:
Tried just switching these to typeof undefined checks since it's simple enough to tack on here and `nomnom` doesn't seem to give a simple way to ban unrecognized arguments; what do you think?

Not sure how to test this easily since existing tests are all oriented around build output...but this does ensure that a `hasOwnProperty` option is ignored instead of causing a TypeError.